spec/features/create_peep_spec.rb

```diff
@@ -0,0 +1,10 @@
+feature 'Creating a new peep' do
```

Comment:
Test coverage is good, but the types of input (try invalids too?) aren't being tested. Only one type of input is being tested here...

Reply:
That's a good point. To make your application more robust, you'd want to make sure you caught invalid inputs and test that too. In the context of this challenge, it's a good enough feature test though :)